### ISSUE TITLE
Map from relative size to relative description

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/ohms-law#76]
@terracoda in #74 we decided to proceed with implementing the relative size descriptions by comparing the relative letter heights in the view. This means that we can define height ratios that map to the comparative descriptions.  Here is how that map works now:

If letter height is less than or equal to 0.25 * height of letter V, it is much much smaller.
If letter height is between 0.25 and 0.5 * height of letter V, it is much smaller.
If letter height is between 0.50 and 0.9 * height of letter V, it is somewhat smaller.
If letter height is between 0.9 and 1.1 * height of letter V, it is comparable.
If letter height is between 1.1 and 2.0 * height of letter V, it is somewhat larger.
If letter height is between 2.0 and 4.0 * height of letter V, it is much larger.
If letter height is > 4.0 * height of letter V, it is much much larger.

Does this mapping seem appropriate to you?  It would probably be easiest to review in the running sim, we will add a dev version with descriptions to play with soon.

**Comment by terracoda** [comment]
@jessegreenberg and @zepumph, this looks great to me. Seven comparative size values should be enough to meet the pedagogical goals of the sim.

Thanks for mapping it out mathematically for me!

**Comment by jessegreenberg** [comment]
Sounds good @terracoda, thanks! Closing.
